### PR TITLE
quickfix

### PR DESCRIPTION
**Pull request by imakiro**
trying to solve https://github.com/dnschneid/crouton/issues/2260 on the advices of @DennisLfromGA

### CODE REVIEW

File: targets/kde [hunk 1]
```diff
@@ -7,9 +7,13 @@ DESCRIPTION='Installs a minimal KDE desktop environment. (Approx. 600MB)'
 HOSTBIN='startkde'
 CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
-
+$TARGET=kde-plasma-desktop
```
Comment (divx118) [inline targets/kde]: This won't work, should be TARGET='kde-plasma-desktop' Also try to use quotes when declaring a string.

File: targets/kde [hunk 2]
```diff
@@ -7,9 +7,13 @@ DESCRIPTION='Installs a minimal KDE desktop environment. (Approx. 600MB)'
...
+#quickfix for vivid package name 
+if [release -lt vivid]; then 
+    $TARGET=plasma-desktop
```
Comment (divx118) [inline targets/kde]: Idem see above

**Comment by divx118** [comment]
Thanks for taking the effort to supply a patch. I am not sure however if we should want to use the global variable TARGET for this. Also IIRC it should be possible to add custom targets for releases when calling the install function.

**Comment by imakiro** [comment]
haem, well first of all i don't use crouton anymore (took the chromebook and flashed it with a real kubuntu), and on the issue note you can see that I was completely lost on what was called where in the crouton script, so I thought that adding an condition here might be the solution.
On vivid, the package name is not 'kde-plasma-desktop' but only 'plasma-desktop' see here : http://packages.ubuntu.com/vivid/kde/plasma-desktop

So ok, I missed the quotes for the name, should I think I undestand that I also proposed the wrong place where to put the code? tell me and I will patch

File: targets/kde [hunk 3]
```diff
@@ -7,9 +7,13 @@ DESCRIPTION='Installs a minimal KDE desktop environment. (Approx. 600MB)'
 HOSTBIN='startkde'
 CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
-
+$TARGET=kde-plasma-desktop
+#quickfix for vivid package name 
+if [release -lt vivid]; then
```
Comment (divx118) [inline targets/kde]: `if release -lt vivid; then`

**Comment by divx118** [comment]
@imakiro No the place is good, indeed $TARGET is used and also capitols are used for global variables. Maybe use something like `installkdedesktop=´kde-plasma-desktop´` Also note when defining a string to a variable you mustn't use $ in front of the variable. Only when you want to expand the variable. I also noted another small error and that is the [] around the check. You should remove them. I added another comment to your code for that.

Just small things :)

**Pull request closed without merging**